### ISSUE TITLE
TODO list

### ISSUE THREAD

**Issue by terremoth** [terremoth/pwa-maker#1]
- [ ] remove pt-br from code and translate it to english
- [ ] increase font-size, bigger than or equal to 16px
- [ ] Missing form labels
- [ ] Orphaned form label
- [x] Contrast errors / pass strict WCAG
- [ ] WAI-ARIA aria attributes
- [ ] Role attributes
